tests/test_fullrank_vi.py

```diff
@@ -0,0 +1,48 @@
+import chex
+import jax
+import jax.numpy as jnp
+import jax.scipy.stats as stats
+import optax
+from absl.testing import absltest
+
+import blackjax
+
+
+class FullRankVITest(chex.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.key = jax.random.PRNGKey(42)
+
+    def test_recover_posterior(self):
+        ground_truth = [
+            # loc, scale
+            (2, 4),
+            (3, 5),
+        ]
+
+        def logdensity_fn(x):
+            logpdf = stats.norm.logpdf(x["x_1"], *ground_truth[0]) + stats.norm.logpdf(
+                x["x_2"], *ground_truth[1]
+            )
+            return jnp.sum(logpdf)
+
+        initial_position = {"x_1": 0.0, "x_2": 0.0}
+
+        num_steps = 50_000
+        num_samples = 500
+
+        optimizer = optax.sgd(1e-2)
+        fullrank_vi = blackjax.fullrank_vi(logdensity_fn, optimizer, num_samples)
+        state = fullrank_vi.init(initial_position)
+
+        rng_key = self.key
+        for _ in range(num_steps):
+            rng_key, _ = jax.random.split(rng_key)
+            state, _ = jax.jit(fullrank_vi.step)(rng_key, state)
```

Comment:
Use chex functionarity instead of jitting by hand here.

Reply:
Could you give an example of how to achieve that? I am not familiar with chex's usage

Reply:
See https://github.com/deepmind/chex#test-variants-variantspy, you can find some example in our test eg https://github.com/blackjax-devs/blackjax/blob/4fa296fd2334991d4aadb8f15096e530449bfdb3/tests/test_integrators.py#L114

Reply:
@junpenglao Done